### DISCUSSION TITLE
研勤AI人臉辨識、圖資技術 大啖智慧城市商機

### DISCUSSION THREAD

**Discussion by ghost** [1101-GS4719-C/Python-Community-GS4719-C#513] [General]
本篇主要講述研勤科技利用人臉辨識等相關AI應用融入地理資訊系統作為物流、金融、保全的地圖服務以及交通、犯罪監視、管制...etc的成果整合，個人覺得若未來智慧城市的構想能越趨成熟與完整，對許多服務業與縣市的市區整合規劃效率肯定能有顯著的提升。
越來越多科技新興公司，也都與人工智能、大數據處理相輔相成，對於不論是國內還是全世界，這些都是值得去期待的，不過過於依賴資訊科技整合一個區域的管理，其實也是需要有相應的資安技術提升就是了

![20220323002250](https://user-images.githubusercontent.com/100221228/159646650-29cb2a2c-b2f7-45fc-8b71-f698a9e8c441.jpg)

https://www.chinatimes.com/realtimenews/20220323002249-260410?fbclid=IwAR2K69BvxKqYPtl1QmKadjRs9HxofkGBI-g29ve2FGsS15XoM_eIUIv48NI&chdtv

**Comment by hapiwangy** [comment]
對於人工智慧漸漸成熟的現在，似乎甚麼事情都可以套上這個技術讓他看起來更上一層樓。但說起來簡單，但做起來卻又不是那麼一回事。期許自己在看著這麼多的實際案例之後自己也能夠可以完整的規劃、實際創作出一套系統，造福社會

**Comment by sean1325418** [comment]
感覺以後警察工作會很輕鬆，電腦可以自動追蹤嫌犯，只要把人抓回來就好

**Comment by xkllkx** [comment]
希望能夠早日普及，給社會大眾一個更安全的社會。

**Comment by HuanYuuuuuu** [comment]
科技技術飛快的進步，同時也會需要相關的規範來限制，不然怕會有有心人士濫用

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望能夠趕快使用。

**Comment by cordydeng** [comment]
這個應用不僅可以讓警察的使用效率大幅提升，還能有效的掌握一個地區的資訊，如果未來全台都能有這種配備，一定能讓大家感到安心

**Comment by Sazmafuji** [comment]
透過AI，皆效率化lol
